### PR TITLE
Update main_pl.ts

### PR DESCRIPTION
**Pull request by artww**
translation update

### CODE REVIEW

File: src/translations/main_pl.ts [hunk 1]
```diff
@@ -465,19 +465,19 @@ OpenSSL może nie być zainstalowane.</translation>
     </message>
     <message>
         <source>Start Thunderbird by clicking on the tray icon if it was closed</source>
-        <translation type="unfinished"></translation>
+        <translation>Uruchom Thunderbird klikając w ikonę na tecce systemowej jeśli został zamknięty.</translation>
```
Comment (Abestanis) [inline src/translations/main_pl.ts:468]:
```suggestion
        <translation>Uruchom Thunderbird klikając w ikonę na tecce systemowej jeśli został zamknięty</translation>
```
In main the settings we generally don't add dots at the end.

File: src/translations/main_pl.ts [hunk 2]
```diff
@@ -465,19 +465,19 @@ OpenSSL może nie być zainstalowane.</translation>
...
     </message>
     <message>
         <source>When detecting a change, wait</source>
-        <translation type="unfinished"></translation>
+        <translation>Jeśli wykryto zmianę, poczekaj</translation>
     </message>
     <message>
         <source> milliseconds</source>
-        <translation type="unfinished"></translation>
+        <translation>milisekund</translation>
```
Comment (Abestanis) [inline src/translations/main_pl.ts:476]:
```suggestion
        <translation> milisekund</translation>
```
There will be a number in front of this, let's add a space like we do for ` second` → ` sekund`.

**Review by Abestanis — CHANGES_REQUESTED** [review]
Thank you very much for updating and contributing to the translations.

I just have two small changes I would like to do before accepting them. Please apply my suggestions if you agree with them.

File: src/translations/main_pl.ts (hunk 2 above)
Comment (artww) [reply src/translations/main_pl.ts:476]: I see three more :

        <source>At startup</source>
        <translation">Przy uruchamianiu</translation>
   
        <source>When hiding Thunderbird</source>
        <translation>Podczas ukrywania Thunderbird</translation>
 
        <source>When showing Thunderbird</source>
        <translation>Podczas wyświetlania Thunderbird</translation>

I upgraded file.

File: src/translations/main_pl.ts [hunk 3]
```diff
@@ -449,35 +449,35 @@ OpenSSL może nie być zainstalowane.</translation>
     </message>
     <message>
         <source>At startup</source>
-        <translation type="unfinished"></translation>
+        <translation">Przy uruchamianiu</translation>
```
Comment (Abestanis) [inline src/translations/main_pl.ts:452]:
```suggestion
        <translation>Przy uruchamianiu</translation>
```

File: src/translations/main_pl.ts [hunk 4]
```diff
@@ -449,35 +449,35 @@ OpenSSL może nie być zainstalowane.</translation>
...
     </message>
     <message>
         <source>When hiding Thunderbird</source>
-        <translation type="unfinished"></translation>
+        <translation>Podczas ukrywania Thunderbird</translation>
     </message>
     <message>
         <source>When showing Thunderbird</source>
-        <translation type="unfinished"></translation>
+        <translation">Podczas wyświetlania Thunderbirda</translation>
```
Comment (Abestanis) [inline src/translations/main_pl.ts:460]:
```suggestion
        <translation>Podczas wyświetlania Thunderbirda</translation>
```

**Review by Abestanis — CHANGES_REQUESTED** [review]

**Review by Abestanis — APPROVED** [review]
Thanks 👍

**Pull request merged**